src/ErrorHandler.php

```diff
@@ -337,4 +297,32 @@ private function cleanBacktraceFromErrorHandlerFrames(array $backtrace, string $
 
         return $cleanedBacktrace;
     }
+
+    /**
+     * @param \ErrorException $errorAsException The error to be passed onto listeners
+     */
+    private function invokeErrorListeners(\ErrorException $errorAsException): void
+    {
+        foreach ($this->errorListeners as $listener) {
+            try {
+                $listener($errorAsException);
+            } catch (\Throwable $exception) {
+                // Do nothing as this should be as transparent as possible
+            }
+        }
+    }
+
+    /**
+     * @param \Throwable $throwable The exception to be passed onto listeners
```

Comment:
Missing description of this method. Does it makes sense to have this code in a separate function (since it's used only one and the only thing it's doing is looping over the listeners)?

Reply:
You're right, it's a reminder of the interfaces that I dropped before.